project-base/storefront/components/Pages/Cart/EmptyCartWrapper.tsx

```diff
@@ -29,8 +29,8 @@ export const EmptyCartWrapper: FC<EmptyCartWrapperProps> = ({
     const [transportAndPaymentUrl] = getInternationalizedStaticUrls(['/order/transport-and-payment'], url);
     const [initiatedLoading, setInitiatedLoading] = useState(false);
     const [isLoadingVisible, setIsLoadingVisible] = useState(true);
-    const isLoading = !currentCart.isInitiallyLoaded || currentCart.isLoading;
-    const isLoginLoading = loginLoading !== 'not-loading';
+    const isLoading = currentCart.isFetching || currentCart.isLoading;
+    const isLoginLoading = !!loginLoading;
```

Comment:
Maybe with the simplification of the condition, this variable is not needed anymore :thinking:

Reply:
Sure, the value seems self descriptive now.